tests/network/utils.py

```diff
@@ -249,15 +249,13 @@ def assert_ssh_alive(ssh_vm, src_ip):
         sleep=1,
         func=run_ssh_commands,
         host=ssh_vm.ssh_exec,
-        commands=[shlex.split("sudo ss -tupan | grep ssh")],
+        commands=[shlex.split("sudo ss -o state established '( sport = 22 )' --no-header")],
```

Comment:
you executing this command thru ssh, so seems `sport = 22` will always give you some output

I think you still need to filter by src_ip in the output, something like:
```suggestion
        commands=[shlex.split("sudo ss -o state established '( sport = 22 )' src {src_ip} --no-header")],
```

Reply:
Done.

Reply:
right, src_ip or dst_ip - depends on where you have initial ssh connections set (connection that you want to be sure still present). 

The main idea of my comment was - without filtering by ip your verification by port only will be always true because you are executing ss command thru the ssh (you connected to port 22 and checked if something is connected to port 22). Here the example: 

```
$ ssh core@10.1.156.2 "sudo ss -o state established '( sport = 22 )' --no-header"
tcp 0      0      10.1.156.2:ssh 10.22.64.16:49468 timer:(keepalive,119min,0)
```